src/components/SearchForm/index.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import { AiOutlineSearch } from 'react-icons/ai';
+
+import styles from './index.module.css';
+
+function SearchForm({ fetchGitRepo }) {
+    const [username, setUsername] = useState('');
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        fetchGitRepo(username);
+        setUsername('');
+    };
+
+    const updateInput = (e) => {
+        const input = e.target.value;
+        setUsername(input);
+    };
+
+    return (
+        <form aria-label="form" onSubmit={handleSubmit} className={styles.form}>
+            <label htmlFor="username"></label>
```

Comment:
Great labels! Form looks perfect

Reply:
Thank you! To be honest maybe I should put there aria-label="search" as Romeo pointed out that it's not really necessary to say a form is a form...